modules/cpmso-yaml-provider-spec-aws.adoc

```diff
@@ -64,8 +76,21 @@ providerSpec:
 <5> Specifies the AWS instance type for the control plane.
 <6> Specifies the cloud provider platform type. Do not change this value.
 <7> Specifies the internal (`int`) and external (`ext`) load balancers for the cluster.
-<8> This parameter is configured in the failure domain, and is shown with an empty value here. If a value specified for this parameter differs from the value in the failure domain, the Operator overwrites it with the value in the failure domain.
+<8> Specifies where to create the control plane instance in AWS.
 <9> Specifies the AWS region for the cluster.
-<10> Specifies the control plane machines security group.
-<11> This parameter is configured in the failure domain, and is shown with an empty value here. If a value specified for this parameter differs from the value in the failure domain, the Operator overwrites it with the value in the failure domain.
-<12> Specifies the control plane user data secret. Do not change this value.
\ No newline at end of file
+<10> This parameter is configured in the failure domain, and is shown with an empty value here. If a value specified for this parameter differs from the value in the failure domain, the Operator overwrites it with the value in the failure domain.
```

Comment:
Should you specify which Operator is performing this action? Maybe it's clear to users embedded in this workflow/content, but it might not hurt to also be more specific if possible.